css-forms-1/Overview.bs

```diff
@@ -695,6 +695,11 @@ input[type=radio]::checkmark {
 ## Selects & buttons
 
 ```css
+select {
+  /* Base appearance select always sizes based on its contents. */
+  field-sizing: content !important;
```

Comment:
!important for now but if we decided to enable the legacy layout (size based on layout of longest option) we could remove that modifier and allow authors to set to fixed to enable it.